Snippets/SqlServer/SqlServer_2.1/NamedConnectionString.cs

```diff
@@ -0,0 +1,42 @@
+﻿    using System.Data.SqlClient;
+    using NServiceBus;
+
+    class NamedConnectionString
+    {
+        void ConnectionString(BusConfiguration busConfiguration)
+        {
+            #region sqlserver-config-connectionstring
+
+            var transport = busConfiguration.UseTransport<SqlServerTransport>();
+            transport.ConnectionString("Data Source=INSTANCE_NAME;Initial Catalog=some_database");
+
+            #endregion
+        }
+
+        void ConnectionName(BusConfiguration busConfiguration)
+        {
+            #region sqlserver-named-connection-string
+
+            var transport = busConfiguration.UseTransport<SqlServerTransport>();
+            transport.ConnectionStringName("MyConnectionString");
+
+            #endregion
+        }
+
+        void ConnectionFactory(BusConfiguration busConfiguration)
+        {
+            #region sqlserver-custom-connection-factory
+
+            var transport = busConfiguration.UseTransport<SqlServerTransport>();
+            transport.UseCustomSqlConnectionFactory(
+                connectionString =>
```

Comment:
No. Why? The purpose of this method is to initialize the connection with any custom declarations (`NOCOUNT` etc).